twisted/internet/base.py

```diff
@@ -601,18 +602,33 @@ def sigInt(self, *args):
         """Handle a SIGINT interrupt.
         """
         log.msg("Received SIGINT, shutting down.")
+        try:
+            import signal
```

Comment:
can we have this implemented in http://twistedmatrix.com/documents/current/api/twisted.internet.posixbase.PosixReactorBase.html ?